Source/Core/Core/HW/WiimoteCommon/WiimoteHid.h

```diff
@@ -21,6 +21,44 @@ constexpr u8 HID_HANDSHAKE_SUCCESS = 0;
 constexpr u8 HID_PARAM_INPUT = 1;
 constexpr u8 HID_PARAM_OUTPUT = 2;
 
+class HIDWiimote
+{
+public:
+  using InterruptCallbackType = std::function<void(u8, const u8* data, u32 size)>;
```

Comment:
Did you intentionally omit the `hid_type` parameter name here?

Reply:
Nope. I'll add it.